docs: document edge host known issue - PE-4638 (#3300)
spectrocloud/librarium: docs/docs-content/release-notes/known-issues.md

| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | --------------------- |

> | When you add a new VMware vSphere Edge host to an Edge cluster, the IP address may fail to be assigned to the Edge host upon its reboot.                                                                                                                                                                                                                                                                                                                                                 | Review the [Edge Troubleshooting](../troubleshooting/edge.md) section for workarounds.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | July 9, 2024   | Edge                  |

Reviewer: ```suggestion
| When you add a new VMware vSphere Edge host to an Edge cluster, the IP address may fail to be assigned to the Edge host after a reboot.                                                                                                                                                                                                                                                                                                                                                 | Review the [Edge Troubleshooting](../troubleshooting/edge.md) section for workarounds.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | July 9, 2024   | Edge                  |
```

Author: :white_check_mark: :)
